dnf/cli/commands/install.py

```diff
@@ -45,7 +45,7 @@ class InstallCommand(commands.Command):
 
     @staticmethod
     def set_argparser(parser):
-        parser.add_argument('package', nargs='+', metavar=_('PACKAGE'),
+        parser.add_argument('package', nargs='*', metavar=_('PACKAGE'),
```

Comment:
Feeling is good, counter example is better. Regarding `--security` and other options that only store true, they will be deleted/not implemented/not supported.